CHANGELOG.md

```diff
@@ -4,7 +4,8 @@
 
 ### Features Added
 
-- Support for Azure Key Vault keys https://github.com/Azure/azure-mcp/pull/119
+- Support for Azure Service Bus Queue. https://github.com/Azure/azure-mcp/pull/144
+- Support for Azure Key Vault keys. https://github.com/Azure/azure-mcp/pull/119
```

Comment:
These would have to go now under [0.0.15](https://github.com/Azure/azure-mcp/blob/main/CHANGELOG.md#0015-unreleased). You'll get the changes for this file if you merge main into your branch.